CHANGELOG.md

```diff
@@ -0,0 +1,72 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## 0.0.1 - 2018-10-31
+### Added
+- Add PPO and DDPG for TensorFlow branch
+- Add HER to DDPG
+- Add recurrent policy to NPO
+- Base class for ReplayBuffer, and two implementations: SimpleReplayBuffer
+  and HerReplayBuffer
+- Sampler classes OffPolicyVectorizedSampler and OnPolicyVectorizedSampler
+- Base class for offline policies OffPolicyRLAlgorithm
+- Benchmark tests for TRPO, PPO and DDPG to compare their performance with
+  those produced by OpenAI baselines
+- Dynamics randomization for MuJoCo environments
+- Support for dm_control environments
+- Add dict support for garage environments
+- PEP8 checks enforced in the codebase
+- Support to keep all imports in python files ordered, removed if they're
+  not required, as well as free of import errors.
+- TravisCI support using Docker
+- Test tree under for all the code developed in garage.
+- Code Coverage measurement with codecov
+- Git hooks to enforce PEP8, and to verify imports and commit messages. They're
+  also applied in the Travis CI verification
+- Docstring verification for added files that are not in the test branch or
+  moved files
+- TensorBoard Support
```

Comment:
support

Reply:
TensorBoard support for all key-value/`log_tabular` calls, plus support for logging distributions